templates/zerver/dev_tools.html

```diff
@@ -0,0 +1,40 @@
+{% extends "zerver/portico.html" %}
+
+{# Login page. #}
+{% block portico_content %}
+<div id="devtools-page">
+    <h2>Useful development URLs</h2>
+    <p>
+        Below is a list of useful URLs that provide information useful in development.
```

Comment:
Well, if you're viewing this page, then your dev server must be running :P
I guess it couldn't hurt to add.